demos/signature/EVP_Signature_demo.c

```diff
@@ -0,0 +1,261 @@
+/*-
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Example of using EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* methods
```

Comment:
An example that uses the EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* methods to calculate and verify a signature of two static buffers.

Reply:
Done in latest upload, thanks for your comments.